packages/plugin-types/API.md

```diff
@@ -135,6 +139,22 @@ export type MainStackNameResolver = {
     resolveMainStackName: () => Promise<string>;
 };
 
+// @public (undocumented)
+export type PackageManagerController = {
+    projectRoot: string;

```

Comment:
can `projectRoot` be private?

Reply:
The only reason we have it public is for the following verbiage:

```js
    const cdPreamble =
      process.cwd() === this.packageManagerController.projectRoot
        ? ''
        : `Navigate to your project directory using
'cd .${this.packageManagerController.projectRoot.replace(process.cwd(), '')}'.
Then get started with the following commands:
`;
```

Reply:
I'd remove it and just plumb `projectRoot` from here https://github.com/aws-amplify/amplify-backend/blob/4b6b0bbcd9fb786059514a87075c6bf2abe2f98a/packages/create-amplify/src/create_amplify.ts#L31-L36 to the place it's used.

I.e. package manager controller shouldn't be used to flow that data into that place in first place. It does need that piece of data internally but there doesn't seem to be a need it exposes that in public api.

Btw. In `PackageManagerControllerFactory` and `PackageManagerController` types I'd rename `projectRoot` to `workingDirectory`/`cwd`. The reason is that if you think about `PackageManagerController` as standalone module the "project root" terminology doesn't make much sense - it's CLI and create-amplify that make that interpretation. For executing package manager commands it's just a "working dir".

Reply:
changed `projectRoot` to private/protected 0225440def

Reply:
rename `projectRoot` to `cwd` 995d05eeea